package com.example.tarea2.dto;

public class UsuarioDto {
    private String nombre = "";
    private String contrasennia = "";

    public UsuarioDto() {
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getContrasennia() {
        return contrasennia;
    }

    public void setContrasennia(String contrasennia) {
        this.contrasennia = contrasennia;
    }
}
